ui/desktop/electron-app/config/desktop.js

```diff
@@ -0,0 +1,63 @@
+const fs = require('fs');
+const path = require('path');
+
+// Create config for use in electron forge
+const createConfig = () => {
+  const configPath = path.join(__dirname, '..', 'config');
+  const macosConfigPath = path.join(configPath, 'macos');
+  const appIconPath = path.join(configPath, 'app-icons', 'icon');
+
+  const config = {
+    name: 'boundary',
+    productName: 'Boundary',
+    bundleId: 'com.electron.boundary',
+    copyright: `Copyright © ${new Date().getFullYear()} HashiCorp, Inc.`,
+    releaseVersion: process.env.RELEASE_VERSION,
+    releaseCommit: process.env.RELEASE_COMMIT,
+    icon: appIconPath,
+    macos: {
+      hardenedRuntime: true,
+      entitlements: path.join(macosConfigPath, 'entitlements.plist'),
+      signatureFlags: 'library',
+      signingIdentity: process.env.BOUNDARY_DESKTOP_SIGNING_IDENTITY,
+      dmg: {
+        diskIcon: path.join(macosConfigPath, 'disk.icns'),
+        background: path.join(macosConfigPath, 'background.png'),
+      },
+    },
+  };
+
+  if (!config.releaseVersion) {
+    config.releaseVersion = '0.0.0';
+    console.warn(
+      `[desktop-config] Release version not specified. Using: ${config.releaseVersion} instead`
+    );
+  }
+
+  console.log(`[desktop-config] Release commit: ${config.releaseCommit}`);
+  console.log(`[desktop-config] Release version: ${config.releaseVersion}`);
+
+  // MacOS signing identity
+  if (!config.macos.signingIdentity)
+    console.warn(
+      '[desktop-config] WARNING: Could not find signing identity. Proceeding without signing.'
```

Comment:
This is just the config file generator.  It should contain almost no logic.  This feels like a forge config concern.

Reply:
This file is generated even during development, which would trigger this warning, correct?

Reply:
Removed signing warning as using developer signed builds is discouraged.